test/fluentd-forward.sh

```diff
@@ -14,7 +14,7 @@ update_current_fluentd() {
 
     # undeploy fluentd
     os::log::debug "$( oc label node --all logging-infra-fluentd- )"
-    os::cmd::try_until_failure "oc get pod $fpod"
+    os::cmd::try_until_failure "oc get pod $fpod" $FLUENTD_WAIT_TIME
```

Comment:
You should take the time to change all of  these to use the comparison I did in `rollout.sh` -- just because a pod exists doesn't mean it's been scheduled, the init containers have fired, it hasn't died, etc...